lib/dialogue-agent/conversation.ts

```diff
@@ -215,6 +224,18 @@ export default class Conversation extends events.EventEmitter {
         return this._history;
     }
 
+    get inTestMode() : boolean {
```

Comment:
I mentioned this on Slack, I don't like "test mode", because it's confusing with running inside unit tests or with a simulator. I would rather "recording mode" or similar.